### ISSUE TITLE
The extension isn't working.

### ISSUE THREAD

**Issue by stevenhtek** [TheGoddessInari/netflix-1080p-firefox#13]
The extension isn't working now. So i disabled and enabled then try again. not working. Then i removed the extension and added again no sign of 1080p. i tested with test pattern while the extension is enabled, the result is only 720p. ctrl+shift+alt+D and see the bitrate is only around 3000 and 720p. Please test the extension again and make it work asap. PLEASE. This extension is very essential and for users who watches netflix on firefox. PLEASE NOTICE AND MAKE IT WORK AGAIN ASAP.

**Comment by TheGoddessInari** [comment]
Sure, as soon as github donations reaches $15/month, I can spend 100%on
Netflix. I'm obviously not going to spend my little food money on an
optional that can't even come close to supporting itself. I've made it
really clear, that's in description & on github.

Demanding and being aggressive about an open source extension is
inappropriate.

Feel free to fix it in the existing style of code used & provide a PR.
Nobody prevents you from doing that, but if I literally can't pay for
Netflix, obviously I can't even download the new app player myself with out
paying $15/month.i am just one, and I've always done this gratis for the
communities & love of open source, care for people I've never met. 🙌🦊

**Comment by forsakenquantum** [comment]
Do you have any thoughts on how to troubleshoot it? I tried getting the newest version of cadmium-player.js and replacing the one in the extension with that new version, and then updating background.js to match the names of the fields in the new version. But while the Ctrl + Alt + S dialog still works, the HD profiles don't seem to (and I think the option to force the maximum bitrate doesn't either).

**Comment by Brandhand** [comment]
There is no problem here,
`Version: 6.0026.945.051`
`UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:86.0) Gecko/20100101 Firefox/86.0`

The new series 'Tribes of Europe' is running just fine in 1080

**Comment by vcunat** [comment]
I can confirm it *not* working anymore for me (same series, here in CZ)
```
UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:85.0) Gecko/20100101 Firefox/85.0
```
(FF 86.0 is scheduled for upstream release in a few days; not sure if that could make a difference here)

**Comment by mikeloeven** [comment]
The extension still works however it looks like netflix added a User Agent check. Overriding user agent to edge re enables 1080P although I am unsure of the stability but it works with test pattern.

**Comment by TheGoddessInari** [comment]
I could update the extension with a User Agent for the specific domains, but I'd definitely need to test to see what does/doesn't work, and what the minimum alteration is. At least, I don't like using a bazooka if a scalpel will do.

**Comment by mikeloeven** [comment]
The UA string I am using for netflix is 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36 Edg/88.0.705.74`

**Comment by vcunat** [comment]
No success for me; it instead breaks playback of everything:
> This title is not available to watch instantly. Please try another title. - 1044

**Comment by mikeloeven** [comment]
This was just the workaround that worked for me. I am also using the Waterfox Fork. So its possible this tweak wont work on Mozilla's official builds

**Comment by Zinq444** [comment]
After editing UA according to Mr.mikeloeven's advice,
I was able to watch it at 1080P.
Thank you, Mr.mikeloeven!
And thank you always to Mr.TheGoddess.
take good care of yourself, everyone.

(I'm Japanese, so I use Google Translate to write it in English.
It may be difficult to read. I'm sorry. )

**Comment by SpatiumPrinceps** [comment]
I confirm @vcunat observation. mikeloeven's UA string did not work, instead its breaking any playback. 

However, the addon only today stopped working **AFTER** updating firefox to 86.0. 
I'm in Germany btw.

**Comment by mikeloeven** [comment]
@SpatiumPrinceps UA string is involved you may need to try several different edge strings until you find one that works with your browser version.

**Comment by enveeed** [comment]
I've noticed that the Chrome equivalent of this plugin (https://github.com/jangxx/netflix-1080p/), after also defaulting to 720p, started working for me (Chromium 88, Linux) when I enabled the "Disable VP9" option in the addon options which apparantly changed the stream to an avchigh one instead. Weirdly enough, after resetting this option it kept working, this time with a VP9 stream instead, but also at 1080p.

I tried to investigate if this has any effect in Firefox using this plugins option to disable VP9, but the stream debug info always shows VP9, no matter what I set it to. Not sure if the option is broken or anything, just something weird I thought might be related to the issue.

**Comment by anton-martinus** [comment]
Thanks @enveeed -- that worked for me as well.  Why it works is a bit of a mystery, but as you suggest there may be a clue lurking there somewhere.

**Comment by TheGoddessInari** [comment]
I reupped Netflix just for this, and...Netflix appears to be doing something really different now. It appears to be ignoring the profiles. I'll dig into this more.

**Comment by mikeloeven** [comment]
The scary thing is how invasive does Netflix's browser scan script need to be to allow it to even figure out whats going on

**Comment by lewisbolaton** [comment]
> I've noticed that the Chrome equivalent of this extension (https://github.com/jangxx/netflix-1080p/), after also defaulting to 720p, started working for me (Chromium 88, Linux) when I enabled the "Disable VP9" option in the addon options which apparantly changed the stream to an avchigh one instead. Weirdly enough, after resetting this option it kept working, this time with a VP9 stream instead, but also at 1080p.

Ditto. I also tried that extension with Chromium 88 (Vivaldi 3.6) in Fedora 33 and **with VP9 disabled** from the extension's settings. It might be an issue with Firefox default decoder settings.

**Comment by TheGoddessInari** [comment]
Finally figured out what was going on code-wise. The profileGroups variable also uses profiles, as referenced at the top of https://github.com/jangxx/netflix-1080p/commit/483561fbdc9d56947cd527748025d023ba39a998.

Overriding both places is sufficient to restore 1080p. Clearing this out now.

**Comment by TheGoddessInari** [comment]
This is fixed in 1.15.3.
